meshtastic/tests/test_mesh_interface.py

```
@@ -2444,18 +2490,24 @@ def _capture_telemetry_send(
+            iface._send_pipeline, "waitForTelemetry", wait_for_telemetry
+        )
 
         iface.sendTelemetry(telemetryType="environment_metrics")
         iface.sendTelemetry(telemetryType="air_quality_metrics")
         iface.sendTelemetry(telemetryType="power_metrics")
         iface.sendTelemetry(telemetryType="local_stats")
         iface.sendTelemetry(telemetryType="device_metrics")
-        with caplog.at_level(logging.WARNING, logger=mesh_interface_module.__name__):
+        with pytest.warns(DeprecationWarning) as record:
             iface.sendTelemetry(telemetryType="invalid")
             iface.sendTelemetry(telemetryType="invalid2")
```

Ruff (0.15.7)

[warning] `pytest.warns()` block should contain a single simple statement

(PT031)